lib/App/index.js

```diff
@@ -315,7 +315,12 @@ class App {
         if(Array.isArray(cards)) {
           for( let i = 0; i < cards.length; i++ ) {
             const card = cards[i];
-            this._validateFlowCard(card, `flow.${type}[${card.id || i}]`);
+
+            if (cards.findIndex(other => other.id === card.id) !== i) {
+              console.warn(`Warning: found multiple Flow card ${type} with the id "${card.id}", all Flow cards should have a unique id.`);
+            }
+
+            this._validateFlowCard(card, `flow.${type}.${card.id}`);
```

Comment:
The second parameter has changed, no `i` and missing `[..]`. Is that intentional?

Reply:
Yea that's intentional. `id` is always defined at this point so the `|| i` should never happen. And while with the index a "keypath" (with `[..]`) makes sense, I feel like it makes less sense with the flow card id.